### ISSUE TITLE
no item in Arduin IDE tools to start  EspExceptionDecoder

### ISSUE THREAD

**Issue by kdbqlb** [me-no-dev/EspExceptionDecoder#27]
I try to install the EspExceptionDecoder in the german Arduino IDE.
What ever Folder I use to unzip the files the tool menu remains empty. 
So I m looking for a detailed screeenshot showing the full path of the Folder an ALL files in the folder.
And please give a screenshot of prferences.txt ( Settings of the IDE )
Thank you

**Comment by me-no-dev** [comment]
You do not need to touch anything other than extract into your [sketches folder]/tools.

**Comment by kdbqlb** [comment]
I tried it on 2 Windows PCs but The Tool Menu remains empty. Therefore I ask for detailed path and screenshot showing all necessary files.
Thank you 
Klaus-Dieter 🚴🏊🏼⛵️⛷🎾😊

**Comment by me-no-dev** [comment]
Not a Windows user, so my screenshots will not do any good to you. Maybe you post a screenshot of what you are trying to do?

**Comment by kdbqlb** [comment]
Hi,

here are the screenshots of my “installation” of EspExceptionDecoder.

But there is no tool in the menu. Why ?

Thanks Klaus-Dieter





 

Von: Me No Dev [mailto:notifications@github.com] 
Gesendet: Montag, 30. Oktober 2017 10:49
An: me-no-dev/EspExceptionDecoder
Cc: kdbqlb; Author
Betreff: Re: [me-no-dev/EspExceptionDecoder] no item in Arduin IDE tools to start EspExceptionDecoder (#27)

 

Not a Windows user, so my screenshots will not do any good to you. Maybe you post a screenshot of what you are trying to do?

**Comment by teprom** [comment]
Me neither, do not appear anything for me on Arduino IDE menu's options.

**Comment by me-no-dev** [comment]
@kdbqlb  i do not see screenshots :)

BTW just did a clean install on my Mac and all is fine.
By any chance you guys are using portable version of the IDE?

**Comment by teprom** [comment]
No, I'm using installed IDE ( PT language ). File "EspExceptionDecoder.jar" extracted to folder "C:\Arduino\tools\EspExceptionDecoder\tool". At Tools meny there isn't anything there. Should I reboot system aswell ?
![untitled](https://user-images.githubusercontent.com/9871675/32300369-1dce72de-bf51-11e7-8549-3d363c1ae897.png)

**Comment by teprom** [comment]
Solved!

I had 2 Arduino folders on my Laptop (I don't know why).
Once copied to the other one, now the option is available there.

Thanks.
